### PR TITLE
folder structure

### PR DESCRIPTION
**Pull request by AdithyaShaj**

### CODE REVIEW

File: src/components/molecules/FilteredCard.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React from "react";
-import CrossIcon from "../assets/cross.svg";
-import CustomButton from "./Button";
-import CustomImage from "./common/customimage";
-import { FilteredCardProps } from "../@types/components/commonTypes";
+import CrossIcon from "../../assets/cross.svg";
```
Comment (sarath-sasi-panickaveetil) [inline src/components/molecules/FilteredCard.tsx]: Follow import order

File: src/components/molecules/globalLoader.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
-import GlobalLoaderIcon from "../assets/loader.gif";
-import CustomImage from "./common/customimage";
-import { LoaderProps } from "../@types/components/commonTypes";
+import GlobalLoaderIcon from "../../assets/loader.gif";
```
Comment (sarath-sasi-panickaveetil) [inline src/components/molecules/globalLoader.tsx]: Follow import order

File: src/components/molecules/menucard.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { Card } from "primereact/card";
```
Comment (thasnim-cm) [inline src/components/molecules/menucard.tsx]: import CustomCard instead of Card

File: src/components/atoms/customcard.tsx
Comment (thasnim-cm) [inline src/components/atoms/customcard.tsx:1]: Please follow naming conventions for all the filenames

**Review by thasnim-cm — APPROVED** [review]
Looks good

**Review by sarath-sasi-panickaveetil — APPROVED** [review]
Looks good to me

**Pull request merged**